### PR TITLE
Bugfix/DS-466 - Fix unseen issues from 1.10.0

### PR DESCRIPTION
**Pull request by malalancette-alithya**
This is a big Pull Request as it is a major restructuration of the code and a lot of fixes to fix the issues that was added in the release 1.10.0.

### CODE REVIEW

File: projects/front-end-library/src/lib/components/input-search/twig/index.twig [hunk 1]
```diff
@@ -1,28 +1,54 @@
...
-    class="btn h-100 m-0 p-0 position-absolute--center-right vui--style-reset w-100 sr-only"
-    type="submit"
-    id="{{ input.id }}"
-    name="{{ input.name }}"
-    placeholder="{{ input.placeholder|default("") }}"
-    value="{{ inputSubmit.value|default("search") }}"
-    {{ inputSubmit.extraAttribute|default(null) }}
-  />
-{% endblock %}
+    <!--
```
Comment (DavidAlit) [inline projects/front-end-library/src/lib/components/input-search/twig/index.twig:40]: In HTML on production

File: projects/front-end-library/src/lib/components/link/twig/next.twig [hunk 2]
```diff
@@ -1,26 +1,43 @@
...
-  }}
-  class="hover-translate pseudo-icon--arrow-right h{{ linkSize|default(
-    "4"
-  ) }} font-weight-{{ linkFontWeight|default(
-    "normal"
-  ) }} d-{{ linkDisplay|default("block") }} {{ linkClass|default("") }}"
-  {{ extraAttribute|default(null) }}
->
-  <span>{{ linkLabel|default("Link to next")|raw }}</span>
+<!-- TODO: Do not seem that this file is used anywhere. Check if it is really the case. -->
```
Comment (DavidAlit) [inline projects/front-end-library/src/lib/components/link/twig/next.twig:1]: HTML comment

**Review by DavidAlit — APPROVED** [review]
HTML Comment

**Review by DavidAlit — DISMISSED** [review]

File: projects/front-end-library/src/lib/components/row-button/twig/index.twig [hunk 3]
```diff
@@ -1,51 +1,69 @@
...
 {% set componentTag = isLink ? "a" : "button" %}
 
 {# Arrays of Classes With Dynamic Elements #}
 {% set rowButtonClasses = [
   "bf-row-button",
   "bf-row-button--" ~ size,
   "d-flex align-items-center w-100 px-3",
-  (size == "large" ? "py-4" : "py-3"),
-  (class|default("")),
+  (size is same as("large") ? "py-4" : "py-3"),
```
Comment (malalancette-alithya) [inline projects/front-end-library/src/lib/components/row-button/twig/index.twig:33]: Create a boolean variable for reusability

**Review by DavidAlit — APPROVED** [review]

**Pull request merged**